site/en/docs/devtools/javascript/index.md

```diff
@@ -20,7 +20,7 @@ debugging.
 
 1.  Click **Open Demo**. The demo opens in a new tab.
 
-    [Open Demo][1]
+    <a href="https://googlechrome.github.io/devtools-samples/debug-js/get-started" target="_blank">Open Demo</a>
```

Comment:
I assume we're using an HTML anchor tag because we want to set `target="_blank"`. Since we use markdown-it-attrs, I think we can also accomplish using `{: attrName=value }`.

> ```md
> [Open Demo][1] {: target=_blank }
> ```

That said, I'm not sure which approach is preferable.

Reply:
I think this is less likely to be eaten in any future transition or migration.